### DISCUSSION TITLE
Spillway Mode - Salt Cell

### DISCUSSION THREAD

**Discussion by seaverd** [tagyoureit/nodejs-poolController#488] [Q&A]
@rstrouse 

Quick question.  When you setup the salt cl2 cell, there is a setting to set the percent output of the salt cell for when you are in pool mode and a second setting for when you are in Spa mode.  Is there anyway to add a setting to have a salt cell output for Spillway mode?   Assume if not, I just need to generate a script or automation to do this.  For my setup I basically would just want the salt cell to run at the same output as when its in pool mode.

Thanks,
Dan

**Comment by rstrouse** [comment]
There is no way to add a setting like that and it should not be needed.  The reason there is a separate setting for the spa is that under long usage periods it would be a lot of chlorine for a small amount of water.

**Comment by seaverd** [comment]
@rstrouse

Let me explain why I could use it.  Pool is configured as a
nixie-sharedbody...so pool and spillover spa are using the same pump, salt
cell heater and filter.  For us, we do not keep the spa heated all the
time. So to keep the water fresh, my schedule goes into spillway mode two
times a day for an hour as my goal is to change the water over in the spa.
I just want the salt cell on during that time so I am generating chlorine.
One thing I have noticed is if pool mode is running and then I click the
switch for spillway...the pool mode stays on and the chlorinator continues
to run at the pool settings.  If both pool and spa mode are off and I click
the spillway mode then the chlorinator does not turn on.  So for me when
running a schedule for spillway the chlorinator does not run.

Dan

**Reply by tagyoureit** [reply]
It is working exactly like it is intended. The pool and spa circuits are what dictate the chlorinate output. Simply keep your pool schedule running and overlay the spillway schedule on top of it (run them both at the same time) and it will have the desired effect.
